include/arc_utilities/dijkstras.hpp

```diff
@@ -670,6 +670,155 @@ namespace arc_dijkstras
             }
     };
 
+    template<typename NodeValueType, typename Allocator=std::allocator<NodeValueType>>
+    class SimpleGraphAstar
+    {
+    protected:
+
+        class AstarNode
+        {
+        protected:
+
+            int64_t index_;
+            int64_t backpointer_;
+            double cost_to_come_;
+            double value_;
+
+        public:
+
+            AstarNode(const int64_t index, const int64_t backpointer, const double cost_to_come, const double value) : index_(index), backpointer_(backpointer), cost_to_come_(cost_to_come), value_(value) {}
+
+            inline int64_t Index() const { return index_; }
```

Comment:
And again GraphIndex here? This may not be needed in either location given a high level descriptive comment somewhere.